lib/chef_apply/action/base.rb

```diff
@@ -139,19 +68,6 @@ def notify(action, *args)
         ChefApply::Log.debug("[#{self.class.name}] Action: #{action}, Action Data: #{args}")
```

Comment:
Do we need to figure something out here once this is split into a gem because we wont have telemeter or ChefApply::Log?

Reply:
Yeah, we'll need to talk that through soon. I can see a few ways around it, or we could consider including telemeter.  TargetHost needs to be included as well.